OpenKeychain/src/test/java/org/sufficientlysecure/keychain/operations/BackupOperationTest.java

```diff
@@ -79,11 +79,11 @@
 @RunWith(KeychainTestRunner.class)
 public class BackupOperationTest {
 
-    static Passphrase mPassphrase = TestingUtils.genPassphrase(true);
+    static Passphrase mPassphrase = TestingUtils.testPassphrase0;
```

Comment:
1. This is unused. Extracting the constants and cleaning up the tests might be a worthwhile but boring task (for another PR?).

Reply:
Ok, I've resolved this.